docs/manual/resource-leak-checker.tex

```diff
@@ -539,6 +541,13 @@
 not possible, the programmer will probably need to suppress a warning and
 then verify the code using a method other than the Resource Leak Checker.
 
+
+\sectionAndLabel{Resource Leak Checker Annotation Inference Algorithm}{resource-leak-checker-inference-algo}
+
+There is a special inference mechanism to infer annotations for the Resource Leak Checker. By default, this mechanism operates when the -Ainfer flag is enabled. To utilize the whole-program inference mechanism outlined in section \ref{whole-program-inference}, you should provide the \<-AenableWPIForRLC> flag.
```

Comment:
This section needs to be clear that the inference algorithm is "under construction" - this PR doesn't contain the whole thing. So, for now, this paragraph should lead with something like "we are developing a special inference algorithm" or similar.

Related, when the paper is mentioned below, this section needs to be clear that only part of the algorithm described in the paper is implemented at the current time.

Reply:
update this paragraph.